File: W1_Basic_OOP/basic-io.java
import java.util.Scanner;
public class Program7 {
public static void main(String[] args) {
Scanner myObj = new Scanner(System.in);
System.out.print("Enter the first number: ");
int a = myObj.nextInt();
System.out.print("Enter the second number: ");
int b = myObj.nextInt();
int sum = a + b;
System.out.println("The sum is "+ sum+ ".");
}
}
// This program takes two numbers as input from the user and prints their sum.

File: Theory/W1_Basic_OOP/CPP/basic-io.c
#include <stdio.h>
int main() {
int a, b, sum;
char str[16];
printf("Enter number 1: ");
scanf("%d", &a);
printf("Enter number 2: ");
scanf("%d", &b);
printf("Enter a string: ");
scanf("%s", str);
sum = a + b;
printf("The sum is %d : %s", sum, str);
return 0;

}